src/plugins/git/createGraph.js

```diff
@@ -196,6 +196,59 @@ class GitGraphCreator {
   }
 }
 
+export type BecomesEdge = {|
+  +from: {|
```

Comment:
the type signature implies it's possible that `from.name !== to.name`. My understanding is that with the current algorithm this is impossible, however in the future (rename detection) that would be a valuable relationship to express. is this correct?

Reply:
confirmed offline that this is the case :)

Reply:
(Can confirm confirmation.)